libs/datasets/timeseries.py

```diff
@@ -1150,6 +1150,26 @@ def drop_regions_without_population(
     return mrts.get_locations_subset(locations_with_population)
 
 
+def _compute_derived_vaccinations_initiated(dataset: MultiRegionDataset):
+    administered = dataset.timeseries[CommonFields.VACCINES_ADMINISTERED]
+    completed = dataset.timeseries[CommonFields.VACCINATIONS_COMPLETED]
+    derived_initiated = administered - completed
```

Comment:
I only want to do this if a region has no data at all for vaccinations_initiated, what's the best way to do that?